### PR TITLE
Fix a couple failing tests

### PR DESCRIPTION
**Pull request by mrvisser**
This fixes a consistent failure that currently exists in export, as well as an intermittent failure I came across in the meantime.

### CODE REVIEW

**Comment by mrvisser** [comment]
There still appear to be issues in OAuth with respect to the updated timestamp, but I can't isolate a case of it still.

File: node_modules/gh-series/tests/util.js [hunk 1]
```diff
@@ -577,14 +577,13 @@ var assertSubscribeSeries = module.exports.assertSubscribeSeries = function(clie
...
             assertGetSeriesEvents(client, id, null, null, null, null, function(events) {
-
                 // Generate a broad window for the user's calendar
                 var start = moment().subtract(30, 'days').format();
                 var end = moment().add(30, 'days').format();
 
                 // If the serie contained events, we generate a window around the serie events
                 // and assert that the events are returned in the user's calendar
-                if (!_.isEmpty(events.results.length)) {
+                if (!_.isEmpty(events.results)) {
```
Comment (simong) [inline node_modules/gh-series/tests/util.js]: I fixed this in one of my branches as well and IIRC there's another spot in this file where we do `isEmpty(... .length)`

Comment (mrvisser) [inline node_modules/gh-series/tests/util.js]: You're right. I'll fix that and I found another spot with bogus start/end handling, so I'll push a commit for that as well.

**Comment by simong** [comment]
The OAuth issue is because to the updatedAt and createdAt values use a different date object when creating the record. This can cause the updatedAt value be 1 millisecond earlier than the createdAt value. There's an [open PR for Sequelize that addresses this](https://github.com/sequelize/sequelize/pull/3733/files).

File: node_modules/gh-series/tests/util.js [hunk 2]
```diff
@@ -577,16 +577,17 @@ var assertSubscribeSeries = module.exports.assertSubscribeSeries = function(clie
...
             assertGetSeriesEvents(client, id, null, null, null, null, function(events) {
-
                 // Generate a broad window for the user's calendar
                 var start = moment().subtract(30, 'days').format();
                 var end = moment().add(30, 'days').format();
 
-                // If the serie contained events, we generate a window around the serie events
+                // If the series contained events, we generate a window around the series events
                 // and assert that the events are returned in the user's calendar
-                if (!_.isEmpty(events.results.length)) {
```
Comment (simong) [inline node_modules/gh-series/tests/util.js]: There is a similar bug on line 650 in this file

**Comment by mrvisser** [comment]
Followed up, and fixed a couple more failures. Back to @simong

File: node_modules/gh-auth/tests/util.js [hunk 3]
```diff
@@ -882,11 +886,13 @@ var assertOauthAccessToken = module.exports.assertOauthAccessToken = TestsUtil.c
                     });
                 },
                 'updatedAt': function(actualUpdatedAt) {
+                    // TODO: Enable this when https://github.com/sequelize/sequelize/pull/3733 is resolved
```
Comment (simong) [inline node_modules/gh-auth/tests/util.js]: It seems like the sequelize guys moved faster than we did. Sequelize 3.1.1 has been released and contains that PR.

**Comment by simong** [comment]
PR looks good, feel free to merge into master when undoing those TODOs

**Pull request merged**